plugin.py

```diff
@@ -7,8 +7,8 @@
 from LSP.plugin.core.protocol import Location
 
 
-server_name = 'metals-sublime'
-settings_file = 'metals-sublime.sublime-settings'
+server_name = 'LSP-metals'
```

Comment:
Is this change (server_name) necessary? It's potentially a breaking change for those who have customized something in their global LSP or project settings.

Reply:
> Is this change (server_name) necessary?

It is mainly to remove all these naming inconsistencies. I believe this variable is used status bar message and server name and doesn't leak to user.

Reply:
The server name is what defined the key to use for LSP client settings.
So if someone, for example, wanted to disable this server in a specific project, he would use:
```
  settings: {
    LSP: {
      metals-sublime: {...}
    }
   }
```
Or something similar (don't remember the exact syntax). This change would be breaking in that case.

Reply:
I think that most of the people are not aware of this feature :) (including me)

Reply:
Just was I thought as well. I think it is safe to rename